riscv/tests/instructions.rs

```diff
@@ -0,0 +1,11 @@
+use compiler::compile_asm_string_temp;
+use riscv::compiler::compile_riscv_asm;
+
+fn run_instruction_test(assembly: &str, name: &str) {
+    // TODO Should we create one powdr asm from all tests or keep them separate?
```

Comment:
Probably separated?

Reply:
The problem is that it takes ages to run the tests in that case.

Reply:
but does it make sense to have them all in a single asm file? how do you guarantee reachability of each test?